cmd/egnkey/README.md

```diff
@@ -0,0 +1,19 @@
+## egnkey
+
+### How to build
+Navigate to [egnkey](../egnkey/) directory and run
+```bash
+   go build
+```
+
+### How to run
+
+To create in a random folder
+```bash
+   ./egnkey generate --key-type ecdsa --num-keys <num_key>
```

Comment:
This is ugly. Egnkey should be installed in GOBIN so that it can be run as any other global bin program

Reply:
Actually this makes me think. Why isn’t this just a subcommand of Eigenlayer cli instead?

Reply:
coz - this is not for operators - this is for AVS devs to test. remember I was telling you how go-ethereum also have tools - I agree this installation is not ideal. I just wanted to create something which can unblock DA key creation. can add it later. I also need to add another funding utility

Reply:
gotttcha. ok I'll accept this PR as soon as readme is updated.